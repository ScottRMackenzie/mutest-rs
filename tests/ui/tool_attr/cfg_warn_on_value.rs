//@ build
//@ stderr
//@ rustc-flags: --check-cfg=cfg()

#![allow(unused)]

#[cfg(mutest = "foo")]
fn foo() {}

#[cfg_attr(mutest = "bar", mutest::skip)]
fn bar() {}
