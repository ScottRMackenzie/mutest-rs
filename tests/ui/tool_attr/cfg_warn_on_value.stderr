warning: unexpected `cfg` condition value: `foo`
 --> tests/ui/tool_attr/cfg_warn_on_value.rs:7:7
  |
7 | #[cfg(mutest = "foo")]
  |       ^^^^^^--------
  |             |
  |             help: remove the value
  |
  = note: no expected value for `mutest`
  = help: to expect this configuration use `--check-cfg=cfg(mutest, values("foo"))`
  = note: see <https://doc.rust-lang.org/nightly/rustc/check-cfg.html> for more information about checking conditional configuration
  = note: `#[warn(unexpected_cfgs)]` on by default

warning: unexpected `cfg` condition value: `bar`
  --> tests/ui/tool_attr/cfg_warn_on_value.rs:10:12
   |
10 | #[cfg_attr(mutest = "bar", mutest::skip)]
   |            ^^^^^^--------
   |                  |
   |                  help: remove the value
   |
   = note: no expected value for `mutest`
   = help: to expect this configuration use `--check-cfg=cfg(mutest, values("bar"))`
   = note: see <https://doc.rust-lang.org/nightly/rustc/check-cfg.html> for more information about checking conditional configuration

warning: 2 warnings emitted

